lib/actions/lookup.js

```diff
@@ -12,7 +11,7 @@ const SALESFORCE_API_VERSION = '39.0';
 module.exports.getMetaModel = async function getMetaModel(configuration) {
```

Comment:
Should we remove `async` before `function` in case of absence of `await` inside of the function declaration?

Reply:
No `async` here is fine. It gives an idea that function return Promise.